lib/features/dashboard/dashboard_screen.dart

```diff
@@ -234,7 +234,7 @@ class _DashboardBodyState extends State<DashboardBody>
               children: const [
                 HomePageView(),
                 InspectionView(),
-                ApplicationView(),
+                ApplicationGuideScreen(),
```

Comment:
Please change this widget name to ApplicationView, and delete to dummy ApplicationView file